src/functions-and-arrays.js

```diff
@@ -1,21 +1,47 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2)
+    return num1;
+  else 
+    return num2;
+
+}
```

Comment:
You can do it without the `else` but this is just a detail. Take a look on the lab solution posted on github.